activesupport/lib/active_support/core_ext/array/grouping.rb

```diff
@@ -113,4 +113,8 @@ def split(value = nil)
       results
     end
   end
+
+  def without(*elements) # :nodoc:
```

Comment:
I’d doc it and say what it does exactly as well as point to the enumerable version with a “this is an optimization of that”.

> On Mar 1, 2015, at 18:18, Todd Bealmear <notifications@github.com> wrote:
> 
> In activesupport/lib/active_support/core_ext/array/grouping.rb:
> 
> > @@ -113,4 +113,8 @@ def split(value = nil)
> >        results
> >      end
> >    end
> > +
> > +  def without(*elements) # :nodoc:
> 
> Should we keep this as nodoc or add separate documentation for this from what's in enumerable.rb?
> 
> —
> Reply to this email directly or view it on GitHub.
>